src/h_vialib/secure/nonce.py

```diff
@@ -0,0 +1,29 @@
+"""Create values we can check we created."""
+
+from uuid import uuid4
+
+from h_vialib.secure.token import SecureToken
+
+
+class RandomSecureNonce(SecureToken):
+    """A random token we can verify we created with an expiry time.
+
+    This token should be different every time you create it.
+    """
+
+    def create(self, expires=None, max_age=None):  # pylint: disable=arguments-differ
+        """Create a random nonce which changes every time."""
+        return super().create(
+            payload={"salt": uuid4().hex}, expires=expires, max_age=max_age
+        )
+
+
+class RepeatableSecureNonce(SecureToken):
+    """A token which we can verify we created with an expiry time.
+
+    This token should be the same for the same expiry time.
+    """
+
+    def create(self, expires=None, max_age=None):  # pylint: disable=arguments-differ
+        """Create a random nonce which is the same for the same expiry."""
+        return super().create({}, expires=expires, max_age=max_age)
```

Comment:
I don't know which we need to sort our NGINX local caching in ViaHTML, so I made both. We can swap them out if they are wrong.

Reply:
So `{}` is just being used effectively as a constant value, here? And I guess for PyJWT it has to be a dict? Makes sense

Reply:
Yeah, the stuff underneith could check for None, but I didn't do that.

Reply:
I think I've just realised what you mean. It turns out JWT are repeatable with the same expiry (I think), so we're just creating an empty one.